courses/utils.py

```diff
@@ -30,6 +33,31 @@ def is_grade_valid(override_grade: float):
     return 0.0 <= override_grade <= 1.0
 
 
+# has a learner passed all the requirements for a certificate?
+def has_earned_program_cert(user, program):
+    passed_courses = Course.objects.filter(
+        in_programs__program=program,
+        courseruns__courserungrade__user=user,
+        courseruns__courserungrade__passed=True,
```

Comment:
This should be checking for certificates instead.

Reply:
fixed